crates/test-utils/src/network.rs

```diff
@@ -138,17 +144,20 @@ async fn start_rpc_gateway(
 pub async fn start_rpc_test_network(
     genesis_config: Option<GenesisConfig>,
 ) -> Result<TestNetwork, anyhow::Error> {
-    start_rpc_test_network_with_fullnode(genesis_config, 0).await
+    start_rpc_test_network_with_fullnode(genesis_config, 0, None, None).await
 }
 
 pub async fn start_rpc_test_network_with_fullnode(
     genesis_config: Option<GenesisConfig>,
     fullnode_count: usize,
+    config_dir: Option<&Path>,
+    port: Option<u16>,
```

Comment:
can we introduce a LocalClusterBuilder sort of thing with fluent interfaces? e.g. 
```
    LocalClusterBuilder::new()
        .with_dir(path)
        .with_fullnode(1)
        .with_rpc_port(port)
        .build
        .await
```

We have been trying to be disciplined in adding optional parameters to these helper functions to avoid them from being less ergonomic.

Reply:
1. we don't need `config_dir` with your new change
2. the above comment still applies, let's make a TODO and create an issue if you don't plan to it in this PR. (and rename `port` to `gateway_port` for example :)